static/docs/commands-reference/destroy.md

```diff
@@ -31,7 +31,7 @@ DVC repository destruction.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
-## Example
+## Example: Default Cache directory
```

Comment:
just keep it simple "Example", or "Examples"

Reply:
but [here](https://dvc.org/doc/commands-reference/commit) we have used this naming convention for example.
and imo, `Example: Default Cache directory` is better than just `Example` because we have an example already for the default case.

Reply:
this is the default case. In the link you mentioned it makes sense because `dvc commit` itself makes sense in certain scenarios. So examples are built around use case for them. It's not the case here. "Default Cache directory" makes me think - is there something specific about it? While actually it's 80% of the cases and it's not worth complicating it here. Just put a good title for the second example.